analytic_mixin_analytic_account/models/analytic_mixin.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2023 Quartile Limited (https://www.quartile.co)
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from odoo import api, fields, models
+
+
+class AnalyticMixin(models.AbstractModel):
+    _inherit = "analytic.mixin"
+
+    analytic_account_ids = fields.Many2many(
+        "account.analytic.account",
+        compute="_compute_analytic_account_ids",
```

Comment:
To handle archived analytic accounts as well.
```suggestion
        compute="_compute_analytic_account_ids",
        context={"active_test": False},
```